messages/helpers_test.go

```diff
@@ -89,7 +89,7 @@ func TestMessages_ExtractCommitHash(t *testing.T) {
 	}
 }
 
-func TestMessages_ExtractProposal(t *testing.T) {
+func TestMessages_ExtractProposal(t *testing.T) { //nolint
```

Comment:
Please resolve these linting errors instead of specifying `//nolint`

Reply:
All linter errors

[9854b9a](https://github.com/madz-lab/go-ibft/pull/10/commits/9854b9a0b9e2a32936917e9439990f7772bd5c97)